tools/plugin-toolkit/scripts/preparePackageJson.ts

```diff
@@ -50,17 +49,25 @@ function removeGdStuff(packageJson: Record<string, any>) {
     packageJson.author = "";
     packageJson.description = "";
 
-    delete packageJson.repository;
-
     const { scripts, devDependencies } = packageJson;
 
-    GdScripts.forEach((script) => {
+    GdScriptsRemove.forEach((script) => {
         delete scripts[script];
     });
 
+    GdScriptsUpdate.forEach((script) => {
```

Comment:
This seems kind of complicated, can we inline the config directly in the package.json instead?